content/livesync/index.textile

```diff
@@ -1,6 +1,45 @@
 ---
 title: What is LiveSync?
+product: livesync
 ---
 
+Livesync keeps your frontend clients and your database consistently in sync. It ensures a smooth  and fast user experience through optimistic updates. LiveSync is built on top of Ably’s global message delivery network, guaranteeing reliable messaging delivery and low latency. 
 
+LiveSync is made up of two key components:
 
+* *Database Connector.* A backend service for broadcasting changes from your database in realtime.
+
+* *Models SDK.* A frontend library that simplifies subscribing to changes in data models and applying optimistic updates, merging them with confirmed updates.
+
+<a href="/images/diagrams/what-is-livesync.png" target="_blank">
+  <img src="/images/diagrams/what-is-livesync.png" style="width: 100%" alt="Livesync diagram">
+</a>
+
+h2(#db-connector). Database Connector
+
+The Database Connector uses the "outbox pattern":https://docs.aws.amazon.com/prescriptive-guidance/latest/cloud-design-patterns/transactional-outbox.html to broadcast application-specific change events from your database to frontend clients via "Ably Channels":/docs/channels. 
+
+When you update the data in your database, you record a change event transactionally in a dedicated "outbox table":https://docs.google.com/document/d/19BJ4oYC3ah4wFH0VS5Qv6qHe3qmRS_PpNLSytTOi9tc/edit#heading=h.r00ywdkts746. The Database Connector detects new changes in the outbox and publishes them as messages on Ably Channels. Using the Models SDK, your client applications subscribe to these change events and update their local state with the new data.
+
+The Database Connector is currently a Docker container image that can run on your own infrastructure. If you want to use an Ably-hosted version, please "reach out":https://docs.google.com/forms/d/e/1FAIpQLSd00n1uxgXWPGvMjKwMVL1UDhFKMeh3bSrP52j9AfXifoU-Pg/viewform.
+
+h2(#models-sdk). Models SDK
+
+The Models SDK allows you to easily create live and observable data models in your frontend applications, ensuring they remain synchronized with your database's state in realtime. By subscribing to a model, you receive the changes made by other users, enabling the creation of a seamless, multiplayer application experience.
+
+The Models SDK works through a series of processes: 
+
+- "Sync":/livesync/sync := a virtual area in your application for realtime collaboration among users.
+- "Merge":/livesync/merge := the most common way of showing the online status of users in an application.
+- "Optimistic updates":/livesync/updates := a method to track user interactions within an application, such as a cell or form field.
+- "Subscribe":/livesync/subscribe := a way to efficiently track the cursor positions of users in realtime.
+
+The Models SDK is standalone, built on Ably’s JavaScript SDK, and can be easily integrated into any front-end framework. There are several advantages when used in conjunction with the Ably JavaScript SDK:
```

Comment:
Should be:

The Models SDK, a standalone SDK, is built on [Ably’s JavaScript SDK](https://github.com/ably/ably-js) and can be easily integrated into any front-end framework.
The Models SDK offers several advantages when used with the Ably JavaScript SDK:

Reply:
Change made in this commit [bdab292](https://github.com/ably/docs/pull/2054/commits/bdab2926e8a795eee31ca16c571adabb3d7b9811)